app/views/providers/savings_and_investments/show.html.erb

```diff
@@ -1,8 +1,11 @@
-<%= page_template page_title: t('.h1-heading') do %>
-<%= render(
-      'shared/forms/savings_and_investments_form',
-      bank_accounts: [],
-      form_path: providers_legal_aid_application_savings_and_investment_path(@legal_aid_application),
-      show_draft: true
-    ) %>
+<%= page_template page_title: t('.h1-heading'), template: :basic, show_errors_for: @form do %>
+    <%= render partial: 'shared/forms/savings_and_investments_form',
+               locals: {
+                 model: @form,
+                 show_draft: true,
+                 bank_accounts: [],
+                 or_break: t('.or_break'),
+                 none_selected: t('.none_selected'),
+                 form_path: providers_legal_aid_application_savings_and_investment_path(@legal_aid_application)
+               } %>
```

Comment:
If you are not passing a collection in, the preferred (as it's more succinct) syntax is:
```ruby
render 'path/to/partial', a: 1, b: 2
```
Rather than:
```ruby
render partial: 'path/to/partial', locals: {a: 1, b: 2}
```

So here:
```ruby
render  'shared/forms/savings_and_investments_form',
            model: @form,
            show_draft: true,
            bank_accounts: [],
            or_break: t('.or_break'),
            none_selected: t('.none_selected'),
            form_path: providers_legal_aid_application_savings_and_investment_path(@legal_aid_application)

Reply:
isn't bank_accounts a collection (when it exists?)

Reply:
The key thing is that you are not needing to pass in a `collection:` at the same level as `locals:`. All the key/value pairs are locals. So the more succinct form is fine.